source/draw/RasterMonochrome.ooc

```diff
@@ -25,7 +25,7 @@ import Image, FloatImage
 import Color
 import Canvas, RasterCanvas
 
-MonochromeRasterCanvas: class extends RasterCanvas {
+MonochromeRasterCanvas: class extends PackedRasterCanvas {
```

Comment:
Yeah I agree, I was just following the existing convention `<Type>RasterCanvas` :) I think both `Raster<Type>Canvas` and `RasterCanvas<Type>` are ok.